vpd-manager/src/manager_main.cpp

```diff
@@ -43,9 +46,40 @@ int main(int, char**)
 
         exit(EXIT_SUCCESS);
     }
-    catch (const std::exception& e)
+    catch (const std::exception& l_ex)
     {
-        std::cerr << e.what() << std::endl;
+        if (typeid(l_ex) == typeid(vpd::JsonException))
+        {
+            vpd::EventLogger::createAsyncPel(
+                vpd::types::ErrorType::JsonFailure,
+                vpd::types::SeverityType::Warning, __FILE__, __FUNCTION__, 0,
```

Comment:
Can we have the severity as informational with a TODO that severity needs to be revisited here?